test/views/SupportingDocViewSpec.scala

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package views
+
+import base.ViewBaseSpec
+import messages.{BaseMessages, SupportingDocMessages}
+import org.jsoup.Jsoup
+import org.jsoup.nodes.Document
+import play.twirl.api.Html
+import views.html.SupportingDocView
+
+class SupportingDocViewSpec extends ViewBaseSpec with BaseMessages {
+
+  private lazy val injectedView: SupportingDocView = app.injector.instanceOf[SupportingDocView]
+
+
+  "Rendering the Supportdoc page" should {
```

Comment:
Need to add a test for the link you need to create. Also can you change the format to reflect what Mac was talking about with testing static content.